CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+## v1.1.0 (2024-03-05)
+
+[Detailed Release Notes](https://github.com/cytomining/pycytominer/releases/tag/v1.1.0)
+
+### Fix
+
+- **build**: fix build versioning
+- Fix Spherize – use SVD, simplify calculations
```

Comment:
```suggestion
- simplify Spherize transform – epsilon to regularize instead of clip, add additional checks (see #320) 
```

To clarify old implementation was not wrong

Reply:
Thanks, will update.